src/engines/html5/media-source/adapters/native-adapter.ts

```diff
@@ -220,6 +220,9 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
     if (Utils.Object.hasPropertyPath(config, 'text.useNativeTextTrack')) {
       adapterConfig.displayTextTrack = Utils.Object.getPropertyPath(config, 'text.useNativeTextTrack');
     }
+    if (adapterConfig.displayTextTrack !== false) {
+      videoElement.classList.add('playkit-native-text');
```

Comment:
do we need to remove the class at some point?
is it possible that it will be added when it already exists?
Also, maybe better to export the class name to a const.

Reply:
classList won't add it again, but yeah, i guess we want to remove it when the adapter is destroyed or reset